website/content/partials/known-issues/ephemeral-loggers-memory-leak.mdx

```diff
@@ -0,0 +1,14 @@
+### Vault is storing references to ephemeral sub-loggers leading to a memory leak
+
+Vault is unexpectedly storing references to ephemeral sub-loggers which prevents them from being cleaned up, leading to
+a memory leak. This impacts many areas of Vault, but primarily Approle authentication methods, and logins in Enterprise.
```

Comment:
I think the approle leak is a much smaller problem then logins. There's probably more besides approle so I don't think it should be singled out here.

```suggestion
a memory leak. This impacts many areas of Vault, but primarily logins in Enterprise.
```

Reply:
Agreed.